File: 2023/day1/part1.py
import os
data = open(os.getcwd() + "/FILEPATH.txt", "r")
total = 0
for x in data.readlines():
    ny_kod = ""
    for y in x:
        if y.isdigit():
            ny_kod += y
    total += int(ny_kod[0] + ny_kod[-1])
print(total)

File: 2023/day1/part1.js
const fs = require('fs');
const path = require('path');
var total = 0;

fs.readFile(path.join(__dirname, 'data.txt'), 'utf8', (err, data) => {
    for (x of data.split('\n')) {
        let new_code = "";
        for (y of x) {
            if (!isNaN(parseFloat(y))) {
                new_code += y;
            }
        }
        total += parseInt(new_code[0] + new_code[new_code.length - 1]);
    }
    console.log(total)
});